public_html/wp-content/mu-plugins/blocks/package.json

```diff
@@ -43,7 +43,7 @@
 	],
 	"scripts": {
 		"start": "cross-env BABEL_ENV=development NODE_ENV=development webpack --watch",
-		"build": "cross-env BABEL_ENV=development NODE_ENV=development webpack",
+		"build": "cross-env BABEL_ENV=production NODE_ENV=production webpack",
```

Comment:
@coreymckrill , @vedanshujain : Is there any reason this was set to `development`? This is a huge performance increase.

Reply:
It was set to development because IIRC it produced more readable digested that way. Now that we have a separate `debug` setting, its not relevant anymore and I am resolving this conversation.